pywt/_extensions/c/wavelets.c

```diff
@@ -234,19 +268,18 @@ Wavelet* wavelet(char name, unsigned int order)
                 return NULL;
             }
 
-            w = blank_wavelet((N == 1) ? 2 * M : 2 * M + 2);
+            w = blank_discrete_wavelet((N == 1) ? 2 * M : 2 * M + 2);
             if(w == NULL) return NULL;
 
-            w->vanishing_moments_psi = order/10;
-            w->vanishing_moments_phi = -1;
-            w->support_width = -1;
-            w->orthogonal = 0;
-            w->biorthogonal = 1;
-            w->symmetry = SYMMETRIC;
-            w->compact_support = 1;
-            w->family_name = "Biorthogonal";
-            w->short_name = "bior";
-
+            w->base.vanishing_moments_psi = order/10;
+            w->base.vanishing_moments_phi = -1;
```

Comment:
This is the only discrete wavelet with `-1` for a vanishing moment. Is there a reason it doesn't make sense to define vanishing moments for the scaling function on bior wavelets?